templates/.rubocop.yml

```diff
@@ -1,2 +1,44 @@
-inherit_from:
-  - https://raw.githubusercontent.com/renuo/rails-application-setup-guide/master/templates/.rubocop_base.yml
+Rails:
+  Enabled: true
+
+Metrics/LineLength:
+  Max: 120
+
+Style/Documentation:
+  Enabled: false
+
+Style/NonNilCheck:
+  IncludeSemanticChanges: true
+
+Style/PredicateName:
+  Exclude:
+    - 'app/serializers/**/*'
```

Comment:
there rings a  🔔 . comes from a current client project where we had to pass has_blabla attributes to the frontend. but rubocop disallows that. here we have an interface and it can be, that the serializer is written in a format, that does not go along with rubocop, right?